src/main/java/net/sf/marineapi/nmea/io/UDPDataReader.java

```diff
@@ -49,25 +49,36 @@
 	}
 
 	@Override
+	/**
+	 * Cleaned up and commented - no functional change.
+     * @author Bob Schwarz
+     * @see <a href="https://github.com/LoadBalanced/marine-api">marina-api fork</a>

```

Comment:
Typo "marina-api". Not sure if these are needed, though, as Git already tracks the origin pretty well.

Reply:
Done